LiftOff-Project/ClientApp/src/AppRoutes.js

```diff
@@ -24,9 +25,9 @@ const AppRoutes = [
     path: '/create-account',
     element: <CreateAccount />
     },
-    {
-        path: '/my-watch-list',
-        element: <MyWatchList />
+  {
+        path: '/user',
```

Comment:
Add this path anywhere else after we change it. Main's navbar is more updated and a link to our user data can be added.